Tasks/DotNetCoreCLI/dotnetcore.ts

```diff
@@ -72,7 +72,7 @@ export class dotNetExe {
             }
             else {
                 var pattern = path.dirname(projectFile) + "/**/publish";
-                var files = ffl.findFiles(pattern, true);
+                var files = nutil.resolveFilterSpec(pattern, "", true, true);
```

Comment:
Giving it as empty bypasses the baseFolder check. And it has to be a string, I can either give a "" or null then.